src/Sylius/Bundle/ApiBundle/Controller/Payment/GetPaymentConfiguration.php

```diff
@@ -38,7 +38,10 @@ public function __construct(
     public function __invoke(Request $request): JsonResponse
     {
         /** @var PaymentInterface|null $payment */
-        $payment = $this->paymentRepository->findOneByOrderToken($request->get('paymentId'), $request->get('id'));
+        $payment = $this->paymentRepository->findOneByOrderToken(
+            $request->attributes->get('paymentId'),
```

Comment:
This could be considered as BC-Break, but I'm fine with that. Can we add a note about it?

Reply:
we have `@experimental` on the entire apiBundle so IMO it isn't BC-break, but ofc I will add a note to apiUpgrade file